### PR TITLE
Added dataSubtype = normalized

### PR DESCRIPTION
**Pull request by amapeters**

### CODE REVIEW

**Comment by karawoo** [comment]
I'm a little worried that this will be confusing. "normalized" isn't mutually exclusive with the other terms here, is it? I wonder if we need a different key. Or maybe this will be a good case for multiple value annotations, when we get those.

**Comment by amapeters** [comment]
@karawoo or @sgosline - please review

**Comment by amapeters** [comment]
@sgosline - do you have an opinion here? We need a way to differentiate between processed, but not normalized vs processed and normalized

**Comment by sgosline** [comment]
This is very use-case specific, so it's difficult to make the call. Do we still use the `dataMatrix` term? AFAIK it was  a legacy.

**Comment by amapeters** [comment]
@sgosline we are not using dataMtrix. 

@karawoo we have dataSubtype - raw and processed. Processed could mean many things, but we don't have a value that differentiates between a counts matrix, and normalized expression (for example). In that case I would use processed and normalized to differentiate

File: synapseAnnotations/data/experimentalData.json [hunk 1]
```diff
@@ -417,6 +417,11 @@
     "columnType": "STRING",
     "maximumSize": 250,
     "enumValues": [
+      {
+        "value": "normalized'
```
Comment (karawoo) [inline synapseAnnotations/data/experimentalData.json]:
```suggestion
        "value": "normalized",
```

File: synapseAnnotations/data/experimentalData.json [hunk 2]
```diff
@@ -417,6 +417,11 @@
...
+        "description": "A data set that is produced as the output of a normalization data transformation."
```
Comment (karawoo) [inline synapseAnnotations/data/experimentalData.json]:
```suggestion
        "description": "A data set that is produced as the output of a normalization data transformation.",
```

**Comment by amapeters** [comment]
@sgosline - can you take a look at this and let us know if you agree with using adding 'normalized' to dataSubtype

**Comment by sgosline** [comment]
I agree with the term `normalized` and fully acknowledge that it's super subjective, as normalization can have it's own `json` full of terms and methods. But for now I think it's sufficient.

**Pull request merged**